blockchain/common/common.go

```diff
@@ -49,17 +39,23 @@ type FluxAggregatorState struct {
 	CanSubmit     bool
 }
 
-type RunlogRequest struct {
-	Params           map[string]string
-	Payment          string
-	RequestId        string
-	CallbackFunction string
-}
+type RunlogRequest map[string]interface{}
```

Comment:
Because all fields here ended up in what was previously the `Params` field anyways, so with the previous struct the values went a lot back-and-forth, and for new blockchain implementations, this list would just grow never ending.